spec/test_app_templates/lib/generators/test_app_generator.rb

```diff
@@ -5,6 +5,14 @@ class TestAppGenerator < Rails::Generators::Base
   # so the following path gets us to /path/to/hyrax/spec/test_app_templates/
   source_root File.expand_path('../../../../spec/test_app_templates/', __FILE__)
 
+  def install_turbolinks
+    gem 'turbolinks', '~> 5'
```

Comment:
The test app generated using Rails 6 and `engine_cart` is missing `turbolinks` by default, and the test app or any app dependent on Hyrax cannot function properly unless it includes `turbolinks`. I tried including `turbolinks` in the hyrax gemspec, but that does not set it up correctly, and leads to the following error:

```
couldn't find file 'turbolinks' with type 'application/javascript'
```